packages/@react-spectrum/dnd/docs/dragAndDrop.mdx

```diff
@@ -0,0 +1,350 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import docs from 'docs:@react-spectrum/dnd';
+import dndDocs from 'docs:@react-types/shared/src/dnd.d.ts';
+import {FunctionAPI, PageDescription, TypeContext, InterfaceType} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Concepts
+keywords: [drag and drop, dnd]
+after_version: 3.0.0
+order: 6
+---
+
+# Drag and Drop
+
+This page describes how to enable drag and drop functionality for the various React Spectrum components that support it.
+
+## Introduction
+
+Drag and drop is an operation performed when a user "grabs" a UI element via their pointing device and drops it on a different location or element.
+Due to its flexibility across a variety of use cases, it has become a common interaction in many modern applications but can come saddled with
+many accessibility issues, including but not limited to keyboard interaction parity and difficulties mimicking drag operations with mouse emulators.
+React Spectrum collection components that support drag and drop operations seek to address these issues by providing full keyboard and screen reader drag and drop
+support.
+
+## Drag and Drop Concepts
+
+Before we dive into how to enable drag and drop in React Spectrum, let's touch briefly on the terminology and concepts of drag and drop. In a drag and drop operation, there is
+a **drag source** and a **drop target**. The drag source is the starting location of your dragged data and the drop target is its intended destination. The dragged data
```

Comment:
I don't think location makes much sense here?
```suggestion
a **drag source** and a **drop target**. The drag source is the data associated with the element where the drag was started and the drop target is its intended destination. The dragged data
```

Reply:
Not quite, in this case the ListView is a drag source, with the items inside being valid draggable items